testing/vcs/test_script_for_py.py

```diff
@@ -0,0 +1,15 @@
+import vcs
+
+list=["boxfill","meshfill","isofill","textcombined","texttable","vector"]
+for obj in list:
+    s=''
+    tc=''
+    if obj == 'textcombined':
+        vcs.createtextcombined('EXAMPLE_tt', 'qa', 'EXAMPLE_tto', '7left')
+        tc="'EXAMPLE_tt','EXAMPLE_tto'"
+    code="s=vcs.get%s(%s)" % (obj, tc)
+    exec(code)
```

Comment:
It's bad form to use exec for... well, anything. I know it's sprinkled throughout the codebase, but you shouldn't pick up bad habits from that :wink: 

Instead, you can use this:

```
s = vcs.creategraphicsmethod(obj, 'test_%s' % obj)
```

which will let you generate an arbitrary graphics method.